src/CheckoutSdk/Payments/IPaymentsClient.cs

```diff
@@ -16,7 +16,7 @@ public interface IPaymentsClient
         /// <param name="paymentRequest">The payment details such as amount and curency.</param>
         /// <param name="cancellationToken">A cancellation token that can be used to cancel the underlying HTTP request.</param>
```

Comment:
Add param documentation for the `IPaymentClient` and and `IApiClient` methods that now have this parameter.
```suggestion
        /// <param name="paymentRequest">The payment details such as amount and currency.</param>
        /// <param name="idempotencyKey">An optional idempotency key for safely retrying payment requests</param>
        /// <param name="cancellationToken">A cancellation token that can be used to cancel the underlying HTTP request.</param>
```